contracts/Forwarder.sol

```diff
@@ -95,6 +97,44 @@ contract Forwarder is ReentrancyGuard, ERC721TokenReceiver {
     return this.onERC721Received.selector;
   }
 
+  function onERC1155Received(
+    address _operator,
+    address _from,
+    uint256 id,
+    uint256 value,
+    bytes calldata data
+  ) external virtual override nonReentrant returns (bytes4) {
+    IERC1155 instance = IERC1155(msg.sender);
+
+    if (autoFlush721) {
+      instance.safeTransferFrom(address(this), parentAddress, id, value, data);
```

Comment:
any checks we could add here? possibly that msg.sender implements ERC165 for ERC1155?

Reply:
fixed